src/app/page.tsx

```diff
@@ -1,145 +1,53 @@
 // SPDX-FileCopyrightText: 2024 PNED G.I.E.
-//
 // SPDX-License-Identifier: Apache-2.0
 
 "use client";
 
 import PageContainer from "@/components/PageContainer";
 import SearchBar from "@/components/Searchbar";
-import Image from "next/image";
-import Link from "next/link";
 import { useSearchParams } from "next/navigation";
-import dna from "../public/dna.jpg";
-import logo from "../public/gdi-logo.png";
-import gdi_visuals from "../public/gdi-visuals.png";
-import office from "../public/office.jpg";
-import { DatasetCounter } from "./DatasetCounter";
-import { PortalStatistics } from "./PortalStatistics";
 
 const HomePage = () => {
   const queryParams = useSearchParams();
+  const applicationTitle = process.env.NEXT_PUBLIC_APPLICATION_TITLE || "WELCOME TO GDI";
```

Comment:
Can you please rename `NEXT_PUBLIC_APPLICATION_TITLE`  to `NEXT_PUBLIC_HOMEPAGE_TITLE` and `NEXT_PUBLIC_APPLICATION_SUBTITLE` to `NEXT_PUBLIC_HOMEPAGE_SUBTITLE`, to avoid umbiguity. 

We already have `NEXT_PUBLIC_SITE_TITLE` and `NEXT_PUBLIC_SITE_DESCRIPTION`, that could mislead people/